### ISSUE TITLE
[BUG] Second host storage disk does not show in UI details

### ISSUE THREAD

**Issue by dnoland1** [harvester/harvester#3242]
**Describe the bug**
Added a second storage disk to a host and doesn't appear in the UI detail page:
<img width="1441" alt="image" src="https://user-images.githubusercontent.com/468462/205786352-cabab717-48c8-4f6f-947b-0d5079120fcf.png">

It does appear in the config view:
<img width="1410" alt="image" src="https://user-images.githubusercontent.com/468462/205786418-428b0ee8-83ed-4db5-b655-419ddd57de1e.png">

Developer tools shows the following JS error in the console:
```
TypeError: e.value.tags is undefined
    Y HarvesterHostDisk.vue:1
    VueJS 40
    t index.vue:106
    u runtime.js:45
    n runtime.js:274
    t runtime.js:97
    Ee harvester-v1.1.1.umd.min.detail.js:1
    o harvester-v1.1.1.umd.min.detail.js:1
    promise callback*Ee harvester-v1.1.1.umd.min.detail.js:1
    o harvester-v1.1.1.umd.min.detail.js:1
    Re harvester-v1.1.1.umd.min.detail.js:1
    Re harvester-v1.1.1.umd.min.detail.js:1
    fetch index.vue:50
    e NuxtJS
    u runtime.js:45
    n runtime.js:274
    t runtime.js:97
    Babel 4
    NuxtJS 3
    VueJS 13
console.js:31:30
    e console.js:31
    VueJS 43
    t index.vue:106
    u runtime.js:45
    n runtime.js:274
    t runtime.js:97
    Ee harvester-v1.1.1.umd.min.detail.js:1
    o harvester-v1.1.1.umd.min.detail.js:1
    (Async: promise callback)
    Ee harvester-v1.1.1.umd.min.detail.js:1
    o harvester-v1.1.1.umd.min.detail.js:1
    Re harvester-v1.1.1.umd.min.detail.js:1
    Re harvester-v1.1.1.umd.min.detail.js:1
    fetch index.vue:50
    e NuxtJS
    u runtime.js:45
    n runtime.js:274
    t runtime.js:97
    Babel 4
    NuxtJS 3
    VueJS 13
```

**To Reproduce**
1. Launch a new Harvester environment with at least one host and a host with at least two physical disks
2. Add the second disk to the host in the Harvester UI

**Expected behavior**
Expect to see the second storage disk in the detail UI.

**Support bundle**
NA

**Environment**
 - Harvester ISO version: v1.1.1 (can also reproduce on v1.1.0)
 - Underlying Infrastructure (e.g. Baremetal with Dell PowerEdge R630): Equinix m3.small.x86

**Comment by irishgordo** [comment]
@dnoland1  
Thanks for opening this :smile:  - 

This seems to be similar to:
https://github.com/harvester/harvester/issues/2990

It seems, that it stays that way unless disk-tags are added to the disk.
Do you notice that if you add disk-tags to the second disk, that the disk then displays in the UI?

**Comment by dnoland1** [comment]
@irishgordo Yes, if I add a disk tag, it appears in the UI:
<img width="1184" alt="image" src="https://user-images.githubusercontent.com/468462/205789970-8ec9e747-8d75-4ea6-b2ff-a984c0903026.png">

**Comment by irishgordo** [comment]
@dnoland1 ,
This has been fixed in Harvester Version:
master-26b4af3f-head

Just validated:
https://github.com/harvester/harvester/issues/2990#issuecomment-1379571182

And the fix will be brought into the v1.1.2 release, so closing this for now :smile: 

![Screenshot from 2023-01-11 14-32-57](https://user-images.githubusercontent.com/5370752/211932664-432ca652-3497-4f58-b450-631e68e951eb.png)
![Screenshot from 2023-01-11 14-32-01](https://user-images.githubusercontent.com/5370752/211932665-eb4925b4-14dd-430c-af7a-7268ea53b07f.png)

cc: @noahgildersleeve

**Comment by dnoland1** [comment]
Great, thanks!
